include/fiction/technology/charge_distribution_surface.hpp

```diff
@@ -788,10 +808,10 @@ class charge_distribution_surface<Lyt, false> : public Lyt
                 continue;
             }
 
-            const auto dist_min =
+            const double dist_min =
```

Comment:
If `dist_min` represents a distance, should we not use a unit type here?

Reply:
True! Thanks!